pkg/combustion/kubernetes_test.go

```diff
@@ -302,3 +403,57 @@ func TestExtractCNI(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigureManifestsInvalidURL(t *testing.T) {
+	// Setup
+	ctx, teardown := setupContext(t)
+	defer teardown()
+
+	ctx.ImageDefinition.Kubernetes.Manifests.URLs = []string{
+		"k8s.io/examples/application/nginx-app.yaml",
+	}
+
+	// Test
+	err := configureManifests(ctx)
+
+	// Verify
+	require.ErrorContains(t, err, "downloading manifests to combustion dir: downloading manifest 'k8s.io/examples/application/nginx-app.yaml': executing request: Get \"k8s.io/examples/application/nginx-app.yaml\": unsupported protocol scheme \"\"")
+}
+
+func TestConfigureKubernetes_InvalidManifestURL(t *testing.T) {
```

Comment:
Feel free to drop this case, I don't see it bringing anything to the table since `configureManifests()` does not exist outside of `configureKubernetes()`.

Reply:
👍